### ISSUE TITLE
Discussion of GPS options

### ISSUE THREAD

**Issue by elbridge** [elbridge/yard_bot#4]
Let's just track conversation here.  If we link your github account to this as a collaborator responsible for this topic, you'll get updates when I post something new.
So I think we need a better understanding of the GPS landscape.  It would be awesome if we could use the NOAA system.  They've got enough stations to probably meet our needs, but I don't know enough to know if the data they provide is sufficient/compatible with the uBLOX module.  
The other one looks like it would probably be more effort to implement.  See http://mr337.com/blog/unmaned-robotic-mower-navigation-part-2/ for a quick pass at implementing that module.

**Comment by elbridge** [comment]
The existing stations might not be the best solution.  There is a subscription required for accessing the Virginia stations for RTK data.  The one out by me is operated by Smartnet North America.   Just getting initial access runs more than the cost of a second RTK unit to use as a base station.  And then there is monthly fee as well.

**Comment by frivas12** [comment]
I agree after more research we are going to need a base.  Here is a link to live map of GPS satellites [https://in-the-sky.org/satmap_worldmap.php](url).  We would need an exist strategy if we loose too much resolution because we have lost too many satellites while mowing out in the middle of the yard, maybe just stop and wait.

The link you have above was written in 20016.  The current NS-HP-GL tracks all GPS and all GLONASS in view satellites.  Check this link out [https://discuss.ardupilot.org/t/rtk-vs-m8n-comparison-rover/31238](url) for comparison.  
I think the both GPS options are comparable but I think your right, we need to see which will require less pain and have more source code.

**Comment by elbridge** [comment]
Very good point.  I'm on board with using a ZED-F9P based board.

**Comment by elbridge** [comment]
Also, antennas matter.
https://www.u-blox.com/sites/default/files/products/documents/GPS-Antenna_AppNote_(GPS-X-08014).pdf
https://www.use-snip.com/kb/knowledge-base/question-what-antenna-should-i-use/

**Comment by elbridge** [comment]
Maybe something like this
https://www.alibaba.com/product-detail/Roadmarker-IP67-GPS-L1-L2-GLONASS_60790565221.html?spm=a2700.7724857.normalList.22.2463106dNjr8RB

**Comment by elbridge** [comment]
This from ublox might be a good compromise for cost vs quality:
https://www.u-blox.com/en/product/ann-mb-series
